boards/beagle/pocketbeagle_2/doc/index.rst

```diff
@@ -0,0 +1,107 @@
+.. zephyr:board:: pocketbeagle_2
+
+Overview
+********
+
+PocketBeagle 2 is a computational platform powered by TI AM62x SoC (there are two
+variants, AM6232 and AM6254).
+
+The board configuration provides support for the ARM Cortex-M4F MCU core and the
+following features:
+
+- Nested Vector Interrupt Controller (NVIC)
+- System Tick System Clock (SYSTICK)
+
+The board configuration also enables support for the semihosting debugging console.
+
+See the `PocketBeagle 2 Product Page`_ for details.
+
+Hardware
+********
+The PocketBeagle 2 features the AM62x SoC, which is composed of a dual or quad
+Cortex-A53 cluster and a single Cortex-M4 core in the MCU domain. Zephyr is
+ported to run on the M4F core and the following listed hardware specifications are
+used:
+
+- Low-power ARM Cortex-M4F
+- Memory
+
+   - 256KB of SRAM
+   - 512MB of DDR4
+
+Supported Features
+==================
+
+.. zephyr:board-supported-hw::
+
+Devices
+========
+System Clock
+------------
+
+This board configuration uses a system clock frequency of 400 MHz.
+
+DDR RAM
+-------
+
+The board has 512MB of DDR RAM available. This board configuration
+allocates Zephyr 4kB of RAM (only for resource table: 0x9CC00000 to 0x9CC00400).
+
+Serial Port
+-----------
+
+This board configuration uses a single serial communication channel with the
+MCU domain UART (MCU_UART0).
+
+SD Card
+*******
+
+Download BeagleBoard.org's official `BeagleBoard Imaging Utility`_ to create bootable
+SD-card with the Linux distro image. This will boot Linux on the A53 application
+cores. These cores will then load the zephyr binary on the M4 core using remoteproc.
+
+The default configuration can be found in
+:zephyr_file:`boards/beagle/pocketbeagle_2/pocketbeagle_2_am6232_m4_defconfig`
+
+Flashing
+********
+
+The board can using remoteproc, and uses the OpenAMP resource table to accomplish this.
+
+The testing requires the binary to be copied to the SD card to allow the A53 cores to load it while booting using remoteproc.
+
+To test the M4F core, we build the :zephyr:code-sample:`hello_world` sample with the following command.
+
+   .. zephyr-app-commands::
+      :board: pocketbeagle_2/am6232/m4
+      :zephyr-app: samples/hello_world
+      :goals: build
```

Comment:
Sorry, there should be no spaces before this directive. I should check the Preview tab next time ;)
```suggestion
.. zephyr-app-commands::
   :board: pocketbeagle_2/am6232/m4
   :zephyr-app: samples/hello_world
   :goals: build
```